<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Acquire Impedance Spectrum</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="graph_data">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>199</width>
      <height>178</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <layout class="QGridLayout" name="parameter_gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Output level [%]</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Reference impedance [Ohm]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="fmin_SpinBox">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.030000000000000</double>
       </property>
       <property name="maximum">
        <double>20.000000000000000</double>
       </property>
       <property name="value">
        <double>0.300000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="average_SpinBox">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="z0_SpinBox">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>50.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="output_SpinBox">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="value">
        <double>50.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Low frequency [MHz]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>High frequency [MHz]</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="np_SpinBox">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1600</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Number of points</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Averaging</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="fmax_SpinBox">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>20.000000000000000</double>
       </property>
       <property name="value">
        <double>20.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>COM7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acquire_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <property name="text">
     <string>Acquire Trace</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtitle_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Trewmac TE300x impedance analyser</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="status_textEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>fmin_SpinBox</tabstop>
  <tabstop>fmax_SpinBox</tabstop>
  <tabstop>np_SpinBox</tabstop>
  <tabstop>average_SpinBox</tabstop>
  <tabstop>output_SpinBox</tabstop>
  <tabstop>z0_SpinBox</tabstop>
  <tabstop>acquire_button</tabstop>
  <tabstop>save_button</tabstop>
  <tabstop>close_button</tabstop>
  <tabstop>graph_data</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
